meta_client_v2/Cargo.toml

```diff
@@ -22,5 +21,6 @@ serde = "1.0"
 serde_derive = "1.0.81"
 serde_json = "1.0.60"
 snafu = { version ="0.6.10", features = ["backtraces"]}
+table_engine = { path = "../table_engine" }
```

Comment:
troublesome working: update `docs/dev/crate-deps.dot` 🙈

Reply:
The `table_engine` is not used in the crate and I will remove it.